packages/core-data/src/resolvers.js

```diff
@@ -103,7 +103,46 @@ export function* getEmbedPreview( url ) {
  * Requests Upload Permissions from the REST API.
  */
 export function* hasUploadPermissions() {
```

Comment:
Should I mark this as deprecated? I'm unsure on what the policy is post 5.0 🙂

Reply:
I don't know either. Might be worth pinging someone on slack. Will hold off approving until the answer is known.

Reply:
cc. @youknowriad

Reply:
It's probably fine leaving this as is where we support both `canUser` and `hasUploadPermissions`. We can always deprecate later.

Reply:
Yes, for now, we should leave those, we can add a deprecation message if needed but without specifying a version. 

This is something we should discuss more in the next #core-js chats.

Reply:
I noted this in [the agenda for the next meeting](url
https://docs.google.com/document/d/1QADtwgPif4-fCKIlS6rPP9CXhGgWZI2wFx8UfcXf7kk/edit).

Reply:
@noisysocks I noticed you `@deprecated` the selector, but the resolver hasn't been deprecated. I guess technically they're the same interface and the resolver wouldn't be called directly, but I wanted to double-check if it was intentional.

Reply:
That's a good point. I think selectors are the public interface here but it wouldn't hurt to have the `@deprecated` attribute in the resolver as well.

Reply:
Done in 4ea46a5d6c211e9614e9d2304cf1563a73394fc3!